src/runtime/lpython_builtin.py

```diff
@@ -2,30 +2,29 @@
 #from sys import exit
 
 
-def ord(s: str) -> i32:
+def ord(s: str) -> i32: # currently supports characters with unicode value between 32 to 126
     """
     Returns an integer representing the Unicode code
     point of a given unicode character. This is the inverse of `chr()`.
     """
-    if s == '0':
-        return 48
-    elif s == '1':
-        return 49
-#    else:
-#        exit(1)
+    if len(s) != 1:
+        return -1 # not a character
+    i: i32
+    for i in range(32, 127):
+        if chr(i) == s:
+            return i
```

Comment:
For now I think this is fine. This can be improved by casting `s` to an integer, once we support it in LPython (maybe we already do).

Reply:
Got it. Thank you for the feedback.